credentials/utopia-birth-certificate-vc-v2/credential.json

```
@@ -1,14 +1,17 @@
 {
   "@context": [
     "https://www.w3.org/ns/credentials/v2",
-    "https://w3id.org/vital-records/v1rc3",
+    "https://w3id.org/vital-records/v1rc4",
     "https://w3id.org/vc/render-method/v2rc1"
   ],
   "type": [
     "VerifiableCredential",
     "BirthCertificateCredential"
   ],
   "issuer": "https://ugh.example/issuer",
+  "name": "Utopia Birth Certificate",
+  "image": "https://examples.vcplayground.org/credentials/utopia-birth-certificate-vc-v2/image.png",
+  "description": "An official information about a person's birth.",
```

Grammar could be improved.
```suggestion
  "description": "Official information about a person's birth.",
```